docs/ros/ros/config/generators.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Generators
+sidebar_label: Generators
+sidebar_position: 2
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Part of the appeal of using the robot YAML configuration file is that it allows us to generate the necessary files
+to operate the robot at runtime. There are 4 types of files that need to be generated: bash shell scripts, URDF description files,
+launch files, and parameter files. Each file type has a generator which will use the Clearpath config to figure out which files need to
```

Comment:
Is "Clearpath config" defined somewhere? I know we use that term a lot internally, but new customers might not understand it. Can we update the wording here?

Reply:
This will be defined more clearly in the robot YAML section. I did add it to the "Terminology" section of ROS 2 Overview.

Reply:
Got it. Can we make "Clearpath Config" a defined term then (uppercase on the first letter of each word with a link to the Terminology section or the yaml section)?